test-e2e/specs/recordSpec.js

```diff
@@ -36,12 +36,32 @@ describe( 'record', function() {
 
 	 /**************** TEST ****************/
 	it( 'subscribes and sets a record', function(done) {
-		clientA.record.getRecord( 'record1' ).subscribe( 'user.firstname', function( firstname ){
-			expect( firstname ).toBe( 'Wolfram' );
-			done();
-		});
+		var clientARecord = clientA.record.getRecord( 'record1' );
+		var clientBRecord = clientB.record.getRecord( 'record1' );
+
+		var immediateSubscribeCalled = false;
+		function immediateSubscribe( data ) {
+			clientARecord.unsubscribe( immediateSubscribe );
+			immediateSubscribeCalled = true;
+			expect( data ).toEqual( {} );
+		}
+		expect( clientARecord.isReady ).toBe( false );
+		clientARecord.subscribe( immediateSubscribe, true );
+		expect( immediateSubscribeCalled ).toBe( false );
+
+		var update = 1;
```

Comment:
this is using a number since there are three states
1 -> undefined
2 -> Wolfram
3+ -> don't care, but didn't unsubscribe